pulser-core/pulser/devices/_device_datacls.py

```diff
@@ -579,6 +579,137 @@ def to_abstract_repr(self) -> str:
         validate_abstract_repr(abstr_dev_str, "device")
         return abstr_dev_str
 
+    def print_specs(self) -> None:
+        """Prints the device specifications."""
+        title = f"{self.name} Specifications"
+        header = ["-" * len(title), title, "-" * len(title)]
+        print("\n".join(header))
+        print(self._specs())
+
+    @property
+    def specs(self) -> str:
+        """Text summarizing the specifications of the device."""
+        return self._specs(for_docs=False)
```

Comment:
I'm curious as to why you, as a user, would like access to this string... Are you perhaps parsing it? I would agree that this quite inconvenient and I would think unnecessary given that you can access the device instance attributes.

Reply:
I need this string in two places:
1. I am using the `logging` module to log all events of each run. The specs of the device is one thing I want to log for future reference if needed.
2. In my script, I have a function which checks and displays many different properties of the device. This includes the specs, but various other things I am computing.

Reply:
Ok, that's clear, thanks for the explanation!